streams/shared/src/main/scala/zio/stream/ZStream.scala

```diff
@@ -3679,16 +3681,28 @@ object ZStream extends ZStreamPlatformSpecificConstructors {
     }
 
   /**
-   * Creates a stream from a subscription to a hub.
+   * Creates a stream from a subscription to a [[zio.ZHub]]. The outer managed
+   * effect describes subscribing to receive messages from the hub while the
+   * inner stream describes taking messages from the hub.
    */
-  def fromHub[R, E, A](hub: ZHub[Nothing, R, Any, E, Nothing, A]): ZStream[R, E, A] =
-    managed(hub.subscribe).flatMap(queue => fromQueue(queue))
+  def fromHub[R, E, A](
```

Comment:
I am onboard with keeping this one the same, and introducing `fromHubManaged` for what is (now) this one here (the one that returns the stream in a managed).

Reply:
Done.